src/aioswitcher/device/tools.py

```diff
@@ -165,6 +169,28 @@ def convert_str_to_devicetype(device_type: str) -> DeviceType:
     return DeviceType.MINI
 
 
+def convert_token_to_packet(token: str) -> Union[str, None]:
```

Comment:
I don't see how this method can return `none`

Reply:
fixed. change to return str.